net/sing-box/Makefile

```diff
@@ -27,7 +28,7 @@ define Package/sing-box
   SECTION:=net
   CATEGORY:=Network
   URL:=https://sing-box.sagernet.org
-  DEPENDS:=$(GO_ARCH_DEPENDS) +ca-bundle +kmod-tun
+  DEPENDS:=$(GO_ARCH_DEPENDS) +ca-bundle
```

Comment:
+1 from me

Reply:
First of all, I think tun inbound is not a typical application scenario on openwrt, it is more suitable for client devices rather than routers. The first thing you should consider on a router is to use tproxy and redirect.

Reply:
It can seriously affect other components that rely on firewall rules to operate.
If users really want to use it, they can totally install it later via opkg or add it manually at compile time.

Reply:
If kmod-tun is added, then kmod-nft-tproxy should also be added.

Reply:
Apparently it's already supported and I'm using it.

https://sing-box.sagernet.org/zh/configuration/inbound/tproxy/